docs/conf.py

```diff
@@ -31,7 +31,7 @@
 import sphinx_rtd_theme
 
 
-# Based on http://read-the-docs.readthedocs.io/en/latest/faq.html#i-get-import-errors-on-libraries-that-depend-on-c-modules
+# Based on https://read-the-docs.readthedocs.io/en/latest/faq.html#i-get-import-errors-on-libraries-that-depend-on-c-modules
```

Comment:
This gives a 404. The longest stem of the URL that works is  https://read-the-docs.readthedocs.io/en/latest/

Reply:
Let's just delete it altogether. It's not a reference in the sense of "giving credit" like we have with the numerical algorithms, it's more for developers to look up how this works and if we can't find how it works, we can just not have it.